test/boost/row_cache_test.cc

```diff
@@ -1179,7 +1193,31 @@ SEASTAR_TEST_CASE(test_update_failure) {
             mt->apply(m);
         }
 
-        // fill all transient memory
+        struct updater_impl : public row_cache::external_updater_impl {
+            mode test_mode;
+            std::vector<bytes> memory;
+
+            updater_impl(mode test_mode) noexcept : test_mode(test_mode) {}
+
+            void allocate() {
+                while (true) {
+                    auto x = bytes(bytes::initialized_later(), 4 * 1024);
+                    memory.emplace_back(std::move(x));
```

Comment:
How long does this take to fail?

Does it work without the seastar memory allocator? Without bringing the machine down? I guess it won't run in debug mode.

Reply:
I didn't measure but it doesn't seem long when run interactively.
And no, it doesn't run in debug mode since it's under `#ifndef SEASTAR_DEFAULT_ALLOCATOR`